### PR TITLE
Update `MozillaDeepSpeechWav2Txt` recipe

### PR DESCRIPTION
**Pull request by this**
This PR upgrades the `deepspeech-gpu` package used in the `data/mozilla_deepspeech_wav2txt.py` recipe to latest v0.10.0a3.

Please note that latest `deepspeech-gpu` only supports upto Python 3.9. Hence, `MozillaDeepSpeechWav2Txt` recipe won't be working with the upcoming H2O Driverless AI 1.11 release. Furthermore, https://github.com/mozilla/DeepSpeech seems to be not maintained anymore. So we don't expect a `deepspeech-gpu` version compatible with Python 3.11.

### CODE REVIEW

File: data/mozilla_deepspeech_wav2txt.py [hunk 1]
```diff
@@ -36,10 +36,10 @@
 except ImportError:
     from pipes import quote
 
-_global_modules_needed_by_name = ["deepspeech-gpu==0.6.1"]
+_global_modules_needed_by_name = ["deepspeech-gpu==0.10.0a3"]
```
Comment (rsujeevan) [inline data/mozilla_deepspeech_wav2txt.py:39]: The latest version is `0.9.3`, and `0.10.0a3` does not exist on PyPI.
https://pypi.org/project/deepspeech-gpu/

Comment (this) [reply data/mozilla_deepspeech_wav2txt.py:39]: `0.10.0a3` is a pre-release https://pypi.org/project/deepspeech-gpu/0.10.0a3/

Comment (rsujeevan) [reply data/mozilla_deepspeech_wav2txt.py:39]: Thanks, I see it now.

**Comment by this** [comment]
Yes, for now we will exclude this recipe.

**Review by rsujeevan — APPROVED** [review]

**Pull request merged**